src/squlearn/expectation_operator/expectation_operator_base.py

```diff
@@ -53,6 +53,33 @@ def num_parameters(self):
         """Number of free parameters in the expectation operator."""
         return 0
 
+    @property
+    def parameter_bounds(self):
+        """Bounds of the free parameters in the expectation operator."""
+        return np.array([[0, 5]] * self.num_parameters)
+
+    def generate_initial_parameters(
+        self, ones: bool = True, seed: Union[int, None] = None
+    ) -> np.ndarray:
+        """
+        Generates random parameters for the expectation operator
+
+        Args:
+            ones (bool): If True, returns an array of ones (default: True)
+            seed (Union[int,None]): Seed for the random number generator
+
+        Return:
+            The randomly generated parameters
+        """
+        if ones:
+            return np.ones(self.num_parameters)
```

Comment:
else:

Reply:
Interestingly, pylint suggests usually the opposit, hence:

```python
if a=b:
     return a

# continue with code 
``` 

instead of 

```python
if a=b:
     return a
else:
     # continue with code 
```

Reply:
Okay. To me, the latter is more readable but if there‘s a convention it makes sense to follow it.